AutomatedTesting/Gem/PythonTests/EditorPythonTestTools/editor_python_test_tools/editor_component/physx_collider.py

```diff
@@ -0,0 +1,318 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+import azlmbr.physics as physics
+import azlmbr.math as math
+
+from editor_python_test_tools.editor_entity_utils import EditorEntity
+from editor_python_test_tools.asset_utils import Asset
+
+COMPONENT_NAME = "PhysX Collider"
+
+class PhysxCollider:
+    """
+    PhysxCollider class is used to interact with the PhysXCollider Entity Componenet and its properties. This class
+        makes it easy for the user to interact with the PhysxCollider in a natural and easy-to-write/maintain way.
```

Comment:
update: PhysxCollider -> PhysXCollider

Reply:
Updated references that weren't using the class name to PhysX Collider.

Should we break casing standards for class names as well?

Reply:
Hmm I think as this is the precedent that's been set across the rest of the engine/editor so far we should be consistent, though I take your point if we were starting from scratch maybe this way would have been better